lib/spack/spack/test/cmd/install.py

```diff
@@ -373,8 +373,12 @@ def just_throw(*args, **kwargs):
         exc_type = getattr(builtins, exc_typename)
         raise exc_type(msg)
 
-    monkeypatch.setattr(spack.package.PackageBase, 'do_install', just_throw)
+    monkeypatch.setattr(spack.installer.PackageInstaller, '_install_task',
+                        just_throw)
 
+    # TODO: Why does junit output capture appear to swallow the exception
+    # TODO: as evidenced by the two failing packages getting tagged as
+    # TODO: installed?
```

Comment:
I wouldn't worry too much about this as the CDash stuff is way richer.  But maybe we should address it better later.

Reply:
Nah I think it’s fine to leave this one